pages/docs/tutorials/streetlights.md

```diff
@@ -196,16 +197,21 @@ EOT`}
 {`npm install mqtt -g`}
 </CodeBlock>
 
-### 4. Send message to your application:
+### 4. Send correct message to your application:
 <CodeBlock language="bash">
 {`mqtt pub -t 'light/measured' -h 'test.mosquitto.org' -m '{"id": 1, "lumens": 3, "sentAt": "2017-06-07T12:34:32.000Z"}'`}
 </CodeBlock>
 
-### 5. Go back to previous terminal and notice that your application logs the message you just sent.
+### 5. Send incorrect message to your application:
+<CodeBlock language="bash">
+{`mqtt pub -t 'light/measured' -h 'test.mosquitto.org' -m '{"id": 1, "lumens": "3", "sentAt": "2017-06-07T12:34:32.000Z"}'`}
```

Comment:
Please correct this example to maybe `lumens: "54"` or something like that, because I couldn't quickly see what was wrong with the message and thought it looked like a correct one 😅

Reply:
but you will see in the logs 🤷🏼 
the point here is to show exactly on the same data but different format, to kinda showcase how simple change makes a difference